tests/ga/test_exthandlers.py

```diff
@@ -29,11 +34,513 @@
 from azurelinuxagent.common.utils.extensionprocessutil import TELEMETRY_MESSAGE_MAX_LEN, format_stdout_stderr, \
     read_output
 from mock import MagicMock
-from azurelinuxagent.common.protocol.wire import WireProtocol
+from azurelinuxagent.common.protocol.wire import WireProtocol, InVMArtifactsProfile
 from azurelinuxagent.ga.exthandlers import parse_ext_status, ExtHandlerInstance, get_exthandlers_handler, \
     ExtCommandEnvVariable
-from tests.tools import AgentTestCase, patch, mock_sleep, clear_singleton_instances
+from azurelinuxagent.common.protocol.extensions_config_retriever import ExtensionsConfigRetriever, \
+    GOAL_STATE_SOURCE_FABRIC, GOAL_STATE_SOURCE_FASTTRACK, _GOAL_STATE_SOURCE_FILE_NAME, _SEQUENCE_NUMBER_FILE_NAME, \
+    _INCARNATION_FILE_NAME
+from tests.protocol.mocks import MockWireClient, MockProtocol
```

Comment:
`MockProtocol` is not being used anywhere either 
(Suggestion: In pycharm if you hover over an unused import, it would give you an option of `Optimize Import`. That would remove all unused imports and also sort the remaining ones which is pretty cool).

Reply:
removed